Analytics/Classes/SEGAnalyticsConfiguration.h

```diff
@@ -61,6 +61,10 @@ typedef NSMutableURLRequest *_Nonnull (^SEGRequestFactory)(NSURL *_Nonnull);
  */
 @property (nonatomic, assign) NSUInteger flushAt;
 
+/**
+ * The amount of time to wait before each tick of the flush timer. Smaller values will make events delivered in a more real-time manner and also use more battery. `30` by default.
+ */
+@property (nonatomic, assign) NSTimeInterval flushInterval;
```

Comment:
Can someone set this to less than  < 0?

Reply:
I actually think something too low (1ms) may also be problematic. We should just say something like "Don't set this below 10seconds".

Reply:
[According to Apple](https://developer.apple.com/documentation/foundation/nstimer/2091889-scheduledtimerwithtimeinterval?language=objc) :
> **interval**
> > The number of seconds between firings of the timer. If interval is less than or equal to 0.0, this method chooses the nonnegative value of 0.1 milliseconds instead.

I added this note : 
> A value smaller than 10 seconds will seriously degrade overall performance.